tests/components/labOrderEntry/LabEntryForm.test.jsx

```diff
@@ -173,6 +175,13 @@ describe('Component: LabEntryForm', () => {
     expect(dispatch).toBeCalled();
   });
 
+  it('should toggle the urgency state of a test', () => {
+    const instance = getComponent().instance();
+    const dispatch = jest.spyOn(props, 'dispatch');
+    instance.handleUrgencyChange();
+    expect(dispatch).toBeCalled();
+  }) 
+
```

Comment:
is there a way you could simulate and assert based on the *DOM* itself instead of calling the component methods directly? this is a good situation to use mount if the dom element that would trigger the `handleUrgencyChange` method is in a child component.

the test-case here doesn't seem to flow with its message.

please refer to [this](https://medium.freecodecamp.org/the-right-way-to-test-react-components-548a4736ab22) when you have time.

Reply:
This method is actually not used in this component(`LabEntryForm`), it is passed down to another component `LabDraftOrder` as a prop. I believe the testing based on the result of the DOM should be done in the test for that component and not this.
This is all the method contains in this component:
```
handleUrgencyChange = (order) => {
    this.props.dispatch(toggleDraftLabOrdersUgency(order));
 }
```
Consequently, testing for if `dispatch` is called should be enough assertion.

Reply:
ok